ale_linters/ansible/ansible_lint.vim

```diff
@@ -43,7 +56,7 @@ endfunction
 
 call ale#linter#Define('ansible', {
 \   'name': 'ansible',
```

Comment:
You can rename this to `ansible_lint`, if and only if you add an alias for the old name. See `aliases` under `:help ale#linter#Define()`.

Reply:
Done